src/_tests/fixtures/44343-pre-travis/_response.json

```diff
@@ -0,0 +1,260 @@
+                "authoredDate": "2020-04-29T15:49:01Z",
+                "committedDate": "2020-04-29T15:49:01Z",
+                "pushedDate": "2020-04-29T15:50:02Z",
+                "abbreviatedOid": "3eaee75",
+                "oid": "3eaee750bbe12a4e0b713f41294268f5adc5ad89",
+                "__typename": "Commit"
+              },
+              "__typename": "PullRequestCommit"
+            },
+            {
+              "commit": {
+                "checkSuites": {
+                  "nodes": [
+                    {
+                      "app": {
+                        "name": "Azure Pipelines",
+                        "__typename": "App"
+                      },
+                      "conclusion": null,
+                      "resourcePath": "/DefinitelyTyped/DefinitelyTyped/commit/7daa7160e7c65f0316a094d531d0ffbb8f4e9e5f/checks?check_suite_id=639447497",
+                      "status": "IN_PROGRESS",
+                      "url": "https://github.com/DefinitelyTyped/DefinitelyTyped/commit/7daa7160e7c65f0316a094d531d0ffbb8f4e9e5f/checks?check_suite_id=639447497",
+                      "__typename": "CheckSuite"
+                    }
+                  ],
+                  "__typename": "CheckSuiteConnection"
+                },
+                "status": {
+                  "state": "SUCCESS",
+                  "contexts": [
+                    {
+                      "state": "SUCCESS",
+                      "description": "The Travis CI build passed",
+                      "creator": {
+                        "login": "sandersn",
+                        "__typename": "User"
+                      },
+                      "targetUrl": "https://travis-ci.org/github/DefinitelyTyped/DefinitelyTyped/builds/681084023?utm_source=github_status&utm_medium=notification",
+                      "__typename": "StatusContext"
+                    },
+                    {
+                      "state": "SUCCESS",
+                      "description": "The Travis CI build passed",
+                      "creator": {
+                        "login": "sandersn",
+                        "__typename": "User"
+                      },
+                      "targetUrl": "https://travis-ci.org/github/DefinitelyTyped/DefinitelyTyped/builds/681083957?utm_source=github_status&utm_medium=notification",
+                      "__typename": "StatusContext"
+                    }
+                  ],
+                  "__typename": "Status"
+                },
+                "authoredDate": "2020-04-29T16:02:14Z",
+                "committedDate": "2020-04-29T16:02:14Z",
+                "pushedDate": "2020-04-29T16:02:30Z",
+                "abbreviatedOid": "7daa716",
+                "oid": "7daa7160e7c65f0316a094d531d0ffbb8f4e9e5f",
+                "__typename": "Commit"
+              },
+              "__typename": "PullRequestCommit"
+            },
+            {
+              "commit": {
+                "checkSuites": {
```

Comment:
Here: this is the freshly pushed commit with no Travis reply yet

Kudos on pipelines for response speed though, faster than GH actions too